### PR TITLE
Don't index feed pages

### LINKED ISSUE #1248: Don't index feed pages on Google
Feed page like `/trending/*`, `/created/*`, `/promoted/*`, `/hot/*` are not too much relevant in search engine result, we should tell Google to not index them. Maybe using a robots.txt file. We need Google to following these page and go through all post but we don't need these page on Google directly.

### PR DESCRIPTION
**Pull request by Sekhmet**
Fixes #1248 

Changes:
- Add `noindex` to feed pages.

### CODE REVIEW

File: src/client/feed/Page.js [hunk 1]
```diff
@@ -57,6 +57,7 @@ class Page extends React.Component {
       <div>
         <Helmet>
           <title>Busy</title>
+          <meta name="robots" content="noindex" />
```
Comment (bonustrack) [inline src/client/feed/Page.js]: Can you use `noindex,follow`, so Google go through sub links?

Comment (Sekhmet) [reply src/client/feed/Page.js]: There is no such rule as `follow`. If you want to disable following links you use `nofollow` - following links is default.

Comment (bonustrack) [reply src/client/feed/Page.js]: I always used `index,follow`, Medium is using it too `view-source:https://medium.com/byteball/byteball-grants-program-906a71b93d3c`. Unless Google said it doesn't matter at all, it's more safe to have it dont you think?

**Comment by bonustrack** [comment]
Does this apply for the homepage aswell @Sekhmet ?

**Comment by Sekhmet** [comment]
Yes, it does. It shouldn't I guess?

**Comment by bonustrack** [comment]
It should not, we want the homepage indexed.

**Pull request merged**